modules/admin_manual/pages/installation/docker/index.adoc

```diff
@@ -3,19 +3,23 @@
 
 ownCloud can be installed using Docker, using
 https://hub.docker.com/r/owncloud/server/tags[the official ownCloud Docker image].
-This official image works standalone (e.g. for a quick evaluation with `docker run -p{std-port-http}:{std-port-http} owncloud/server`)
-but it is designed to work with a data volume in
-the host filesystem and with separate _MariaDB_ and _Redis_ containers.
+This official image works standalone for a quick evaluation but is designed to be used in a docker-compose setup.
```

Comment:
no idea why you are changing this .... let people run a stand alone container. What is the problem?

Reply:
Problem was without `-e DOCKER_DOMAIN=loclahost:8080` it would not fully work. -- I wanted to add that to make it easier to run the standlone container.

Reply:
Well - then let's just add this argument and we are fine. No need to change the meaning of the rest of the docs.

Reply:
The meaning of the rest is unchanged. I removed duplicated stuff and disentangled the language. I fail to see where this is bad. I can split it in two PRs if needed.

Reply:
plz split it up. THX

Reply:
Done. split into #2858 and #2859